### PR TITLE
More persons BDLaethb5

### PR DESCRIPTION
**Pull request by thea-m**
PRS13107GabraKr was created erroneously and will be deleted immediately after merging (but has to be merged due to the new workflow for deleting records https://betamasaheft.eu/Guidelines/?id=create)

### CODE REVIEW

File: new/PRS13101ZamanfasQ.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            <change who="DR" when="2020-07-21">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ዘመንፈስ፡ ቅዱስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Zamanfas Qǝddus</persName>
+                    <floruit notBefore="1711" notAfter="1716"/>
```
Comment (MassimoVilla) [inline new/PRS13101ZamanfasQ.xml:50]: This is perhaps a silly remark, but doesn't this mean that Zamanfas Qǝddus was born in/after 1711 and died in/before 1716?

File: new/PRS13115SelaKr.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <change who="DR" when="2020-07-29">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ሥዕለ፡ ክርስቶስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Śǝʿla Krǝstos</persName>
+                    <floruit notBefore="1711" notAfter="1716"/>
```
Comment (MassimoVilla) [inline new/PRS13115SelaKr.xml:50]: The same remark as above.

**Comment by thea-m** [comment]
This is the production date of the manuscript, so the only period in which we can arguably place them. The TEI definition of the floruit element is "contains information about a person's period of activity", I thought that it could be used that way. What do you think?

**Review by SusanneHummel — APPROVED** [review]

**Comment by thea-m** [comment]
@MassimoVilla can this be merged?

**Review by MassimoVilla — APPROVED** [review]

**Pull request merged**